minio/thread_pool.py

```diff
@@ -37,12 +37,13 @@ def __init__(self, tasks_queue, results_queue, exceptions_queue):
         self.tasks_queue = tasks_queue
         self.results_queue = results_queue
         self.exceptions_queue = exceptions_queue
-        self.daemon = True
+        self.daemon = False
```

Comment:
Why do we turn off daemon mode, `self.daemon = False` ?

Reply:
It depends on your situation. In our use case, we want to close the thread immediately after the program stopped and it's not important for us to miss an uploading file. I change this flag to its previous value (True) but It's better to have a config that user can change this flag depends on their situation when using py-mino.
tnx

Reply:
@nnehdi,
Thank you very much for working on this fix.
Are you going to push these review comment changes, so that we can merge and close this PR?